cmd/zetaclientd/init.go

```diff
@@ -1,13 +1,25 @@
 package main
 
 import (
+	"encoding/json"
+	"os"
+	"path"
+
 	"github.com/rs/zerolog"
 	"github.com/spf13/cobra"
 
 	"github.com/zeta-chain/zetacore/zetaclient/config"
 	"github.com/zeta-chain/zetacore/zetaclient/testutils"
 )
 
+// solanaTestKey is a local test private key for Solana
+var solanaTestKey = []uint8{
```

Comment:
I assume it is temporary?
Can we track TODOs for future refactoring to make and attach link to issues

Reply:
I'll attache this: https://github.com/zeta-chain/node/issues/2614